Missionframework/modules/00_init/fnc/fn_init_settings.sqf

```diff
@@ -0,0 +1,396 @@
+// Default: true
+[
+    "KPLIB_param_debug",
+    "CHECKBOX",
+    [localize "STR_KPLIB_SETTINGS_GENERAL_DEBUG", localize "STR_KPLIB_SETTINGS_GENERAL_DEBUG_TT"],
+    localize "STR_KPLIB_SETTINGS_GENERAL",
+    true,
+    1,
+    {}
+] call CBA_Settings_fnc_init;
+
+// KPLIB_param_saveInterval
+// Interval of periodic saves in seconds.
+// Default: 60
+[
+    "KPLIB_param_saveInterval",
+    "SLIDER",
+    [localize "STR_KPLIB_SETTINGS_GENERAL_SAVEINT", localize "STR_KPLIB_SETTINGS_GENERAL_SAVEINT_TT"],
+    localize "STR_KPLIB_SETTINGS_GENERAL",
+    [30, 600, 60, 0],
+    1,
+    {}
+] call CBA_Settings_fnc_init;
+
+// KPLIB_param_sectorCap
+// The amount of sectors which can be active at the same time.
+// Default: 6
+[
+    "KPLIB_param_sectorCap",
+    "SLIDER",
+    [localize "STR_KPLIB_SETTINGS_GENERAL_SECCAP", localize "STR_KPLIB_SETTINGS_GENERAL_SECCAP_TT"],
+    localize "STR_KPLIB_SETTINGS_GENERAL",
+    [1, 12, 6, 0],
+    1,
+    {}
+] call CBA_Settings_fnc_init;
+
+// KPLIB_param_stamina
+// Enables/Disables the BI stamina system. (doesn't affect ACE Advanced Fatigue)
+// Default: true
+[
+    "KPLIB_param_stamina",
+    "CHECKBOX",
+    [localize "STR_KPLIB_SETTINGS_GENERAL_STAMINA", localize "STR_KPLIB_SETTINGS_GENERAL_STAMINA_TT"],
+    localize "STR_KPLIB_SETTINGS_GENERAL",
+    true,
+    1,
+    {}
+] call CBA_Settings_fnc_init;
+
+// KPLIB_param_clearVehicleCargo
+// Enables/Disables if spawned vehicles will have an empty cargo space.
+// Default: true
+[
+    "KPLIB_param_clearVehicleCargo",
+    "CHECKBOX",
+    [localize "STR_KPLIB_SETTINGS_GENERAL_CLEARCARGO", localize "STR_KPLIB_SETTINGS_GENERAL_CLEARCARGO_TT"],
+    localize "STR_KPLIB_SETTINGS_GENERAL",
+    true,
+    1,
+    {}
+] call CBA_Settings_fnc_init;
+
+// KPLIB_param_fobRange
+// Build radius in meters around the FOB area center position.
+// Default: 125 meters
+[
+    "KPLIB_param_fobRange",
+    "SLIDER",
+    [localize "STR_KPLIB_SETTINGS_GENERAL_FOBRANGE", localize "STR_KPLIB_SETTINGS_GENERAL_FOBRANGE_TT"],
+    localize "STR_KPLIB_SETTINGS_GENERAL",
+    [100, 250, 125, 0],
+    1,
+    {}
+] call CBA_Settings_fnc_init;
+
+// KPLIB_param_sectorActRange
+// Radius in meters around the sector center to activate the sector.
+// Default: 1200 meters
+[
+    "KPLIB_param_sectorActRange",
+    "SLIDER",
+    [localize "STR_KPLIB_SETTINGS_GENERAL_SECACT", localize "STR_KPLIB_SETTINGS_GENERAL_SECACT_TT"],
+    localize "STR_KPLIB_SETTINGS_GENERAL",
+    [600, 1800, 1200, 0],
+    1,
+    {}
+] call CBA_Settings_fnc_init;
+
+// KPLIB_param_sectorCapRange
+// Radius in meters around the sector center a unit has to be to being able to capture the sector.
+// Default: 150 meters
+[
+    "KPLIB_param_sectorCapRange",
+    "SLIDER",
+    [localize "STR_KPLIB_SETTINGS_GENERAL_SECRANGE", localize "STR_KPLIB_SETTINGS_GENERAL_SECRANGE_TT"],
+    localize "STR_KPLIB_SETTINGS_GENERAL",
+    [100, 200, 150, 0],
+    1,
+    {}
+] call CBA_Settings_fnc_init;
```

Comment:
Sector handling is in core, shouldn't these settings be in core too then?

Reply:
I've also thought about that, but came to the opinion that init/common/core are all able to be categorized as "general settings" and therefore I kept this category together in one file.

Reply:
I'm not sure. If in the future you will want to edit something related to these settings, will you remember that despite that the code for sectors is in `core` the settings related to it are in `init` module?